samples/tfm/tfm_hello_world/sample.yaml

```diff
@@ -0,0 +1,16 @@
+sample:
+  description: Hello World sample, the simplest Zephyr
+    application, with TF-M enabled
+  name: hello world TFM
+common:
+    tags: tfm
+    platform_allow: nrf5340pdk_nrf5340_cpuappns nrf9160dk_nrf9160ns
```

Comment:
Do we need _dk_ here instead of _pdk_? or both...

Reply:
Thanks, added both.